cypress/README.md

```diff
@@ -35,10 +33,9 @@ cy.getContent()
 
 ### Commands from `@nextcloud/cypress`
 
-We also use some commands from `@nextcloud/cypress`:
+We also use some commands from [`@nextcloud/cypress`](https://github.com/nextcloud/nextcloud-cypress):
 * `login(user)`
 * `logout()`
 * `createRandomUser().then(user => login(user))`
 * `createUser(user)`
```

Comment:
Could you add the two commands here?
I think we should either keep this list complete or remove it entirely.
```suggestion
We also use some commands from [`@nextcloud/cypress`](https://github.com/nextcloud/nextcloud-cypress):
* `login(user)`
* `logout()`
* `createRandomUser().then(user => login(user))`
* `createUser(user)`
* `modifyUser(user, key, value)`
* `deleteUser(user)`
```

Reply:
I would vote for removing and only keep a reference of our own commands + a note about the nextcloud package.
(so keep the table but remove the list and only keep `We also use some commands from [`@nextcloud/cypress`](https://github.com/nextcloud/nextcloud-cypress).`

Reply:
Fine with me.